### PR TITLE
HARP-5144: Add picking for omv-datasource

### PR DESCRIPTION
**Pull request by pit-rpg**
HARP-5144, HARP-6565, Picking implemented for OMV data source,
fixed issues with road and label picking.

### CODE REVIEW

File: @here/harp-datasource-protocol/lib/TileInfo.ts [hunk 1]
```diff
@@ -19,6 +19,15 @@ export interface TileInfo {
     readonly numBytes: number;
 }
 
+/**
+ * Represents a feature group type for tile info.
+ */
+export enum FeatureGropeType {
```
Comment (StefanDachwitz) [inline @here/harp-datasource-protocol/lib/TileInfo.ts]: What is a "Grope" ?

File: @here/harp-mapview/lib/PickHandler.ts [hunk 2]
```diff
@@ -137,8 +134,8 @@ export class PickHandler {
      */
     intersectMapObjects(x: number, y: number): PickResult[] {
         const worldPos = this.mapView.getNormalizedScreenCoordinates(x, y);
-        const rayCaster = this.m_rayCaster;
 
+        const rayCaster = this.mapView.raycasterFromScreenPoint(x, y);
```
Comment (StefanDachwitz) [inline @here/harp-mapview/lib/PickHandler.ts]: Why is it necessary to create a new raycaster on every call?

File: @here/harp-mapview/lib/PickHandler.ts [hunk 3]
```diff
@@ -102,7 +101,6 @@ export interface PickResult {
  * Handles the picking of scene geometry and roads.
  */
 export class PickHandler {
-    private readonly m_rayCaster: PickingRaycaster;
     private readonly m_plane = new THREE.Plane(new THREE.Vector3(0, 0, 1));
```
Comment (StefanDachwitz) [inline @here/harp-mapview/lib/PickHandler.ts]: If PickingRayCaster is no longer required, please remove file.

Comment (pit-rpg) [reply @here/harp-mapview/lib/PickHandler.ts]: The PickingRaycaster is used for MapViewPoints that used for Circles and Squares classes. They are almost unused in the code but I don't really sure that I can remove them.

File: @here/harp-mapview/lib/PickHandler.ts (hunk 2 above)
Comment (pit-rpg) [reply @here/harp-mapview/lib/PickHandler.ts]: here the raycasterFromScreenPoint method returns only a link to the MapView.m_raycaster object

**Pull request merged**